### PR TITLE
Update v4 release to 4.67

### PR DESCRIPTION
**Pull request by satvu**
Move 4.67.0 from pre-release to GA.

### CODE REVIEW

**Comment by satvu** [comment]
Blocking this until VS Code issues are resolved

**Review by mattchenderson — APPROVED** [review]
Approving pending closure on the VS Code topic.

**Comment by satvu** [comment]
VS Code PR to verify new templates has been merged [here](https://github.com/microsoft/vscode-azurefunctions/pull/3946).

**Pull request merged**
